src/components/ComponentDoc/ComponentDoc.sass

```diff
@@ -1 +1,64 @@
-@import '../../styles/core'
\ No newline at end of file
+@import '../../styles/core'
+
+section
+    hr
+        border-bottom: 1px solid black
+    
+    h2
+        font-size: 2.5rem
+        line-height: 4rem
+        font-weight: 400
+
+    h3
+        font-size: 2rem
+        line-height: 4rem
+        font-weight: 400
+
+    table
+        font-size: 1.5rem
+        font-weight: 400
+        display: flex
+        flex-direction: column
+        width: 100%
+
+    thead
+        width: 100%
+        display:block
+    tbody
+        display:block
+
+    tr
+        display: flex
+        flex-direction: row
+        justify-content: space-between
+        border-bottom: solid 1px $light-gray
+        margin-bottom: 5px
+    th, td
+        width: 25%
+        text-align: start
```

Comment:
I don't think this is a valid css value. Do you mean `align-items: flex-start` or `text-align: left`?